Plan/common/src/main/resources/assets/plan/locale/locale_FR.txt

```diff
@@ -352,7 +353,9 @@ Version FAIL - Read versions.txt                || Les informations de la versio
 Web User Listing                                ||   §2${0} §7: §f${1}
 WebServer - Notify HTTP                         || Serveur Web : Aucun certificat -> Utilisation du serveur HTTP pour la visualisation.
 WebServer - Notify HTTP User Auth               || Serveur Web : Authentification utilisateur désactivée ! (Non sécurisée avec HTTP)
+WebServer - Notify HTTPS User Auth              || Serveur Web : User Authorization Disabled! (Disabled in config)
```

Comment:
"Serveur Web : User Authorization Disabled! (Disabled in config)" -> "Serveur Web : Authentification d'utilisateur désactivée ! (dans la configuration)"